Packs/GenianNAC/Integrations/GenianNAC/GenianNAC.yml

```diff
@@ -1,10 +1,10 @@
 commonfields:
-  id: Genians
+  id: Genian NAC
```

Comment:
please revert the id filed, you can change the display and the name only

Reply:
I will revert it back.